### PR TITLE
Added react-styling to FAQ

### PR DESCRIPTION
**Pull request by halt-hammerzeit**
The corresponding issue:
https://github.com/FormidableLabs/radium/issues/162#issuecomment-121193028

### CODE REVIEW

File: docs/faq/README.md [hunk 1]
```diff
@@ -86,3 +87,96 @@ To get the latest version, drop this into the HTML:
...
+      <button style={style.button[this.props.kind]}>
+        {this.props.children}
+      </button>
+    )
+  }
+}
+
+const style = styler
+`
+  button
```
Comment (ianobermiller) [inline docs/faq/README.md]: Use regular CSS syntax with the first example (with braces and semicolons) so people aren't too confused

File: docs/faq/README.md [hunk 2]
```diff
@@ -86,3 +87,96 @@ To get the latest version, drop this into the HTML:
 ```
 
 We also recommend changing the JavaScript language to ES6/Babel.
+
+## Can I use my favourite CSS/LESS/SASS syntax?
+
+Yes, you can, with the help of the [react-styling](https://github.com/halt-hammerzeit/react-styling) module. It requires you to write your code in ES6/Babel (everyone does it nowadays). Using [react-styling](https://github.com/halt-hammerzeit/react-styling) module you can write your styles in any syntax you like (curly braces or tabs, CSS, LESS/SASS, anything will do). Below are some examples of how can standard Radium examples be rewritten using this module.
```
Comment (ianobermiller) [inline docs/faq/README.md]: Slight edits:

```
Yes, with the help of the [react-styling](https://github.com/halt-hammerzeit/react-styling) module, which requires [template strings](https://babeljs.io/docs/learn-es2015/#template-strings). Using react-styling you can write your styles in any syntax you like (curly braces or tabs, CSS, LESS/SASS, anything will do). Below are some examples of how can standard Radium examples be rewritten using this module.
```

Yes, with the help of the [react-styling](https://github.com/halt-hammerzeit/react-styling) module, which requires [template strings](https://babeljs.io/docs/learn-es2015/#template-strings). Using react-styling you can write your styles in any syntax you like (curly braces or tabs, CSS, LESS/SASS, anything will do). Below are some examples of how can standard Radium examples be rewritten using this module.

**Comment by halt-hammerzeit** [comment]
@ianobermiller Ok, I've made the changes and repushed. If I missed something let me know.

**Comment by halt-hammerzeit** [comment]
@ianobermiller By the way, I've just implemented support for quotes in CSS syntax style values. So I can add quotes to the first CSS example if you'd like it better.

**Comment by ianobermiller** [comment]
Why would you need quotes in CSS syntax style values? Can you show an example?

How come &:first require an ampersand, but :hover doesn't?

**Comment by ianobermiller** [comment]
Let's remove the second example to keep it concise, and just link to react-styling for more examples and information. Also, can you update the comparison doc to indicate that radium supports CSS syntax when using react-styling?

**Comment by halt-hammerzeit** [comment]
@ianobermiller made the push: faq and comparision

**Pull request merged**